app/src/main/java/com/epfl/drawyourpath/mainpage/fragments/helperClasses/FriendsListAdapter.kt

```diff
@@ -0,0 +1,75 @@
+package com.epfl.drawyourpath.mainpage.fragments.helperClasses
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Button
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.epfl.drawyourpath.R
+
+data class Friend(
+    val id: Int,
+    val name: String,
+    val profileImage: Int, // Use a drawable resource ID for simplicity.

```

Comment:
To store an image you can also used the bitmap image. At t this time, the image is encoded as a bitmap in the user model.

Reply:
I think ill leave it as is for now since it's easier to display the placeholder, but I added a comment that suggests it should be changed in the future when we implement showing the actual profile picture.